src/Pate/Proof.hs

```diff
@@ -109,32 +108,23 @@ import qualified What4.Interface as W4
 -- proof objects
 
 
--- | The body of an 'EquivTriple'.
-data EquivTripleBody sym arch where
-  EquivTripleBody ::
+-- | An triple representing the equivalence conditions for a block pair. Abstracted
+-- over the initial machine state. This is analogous to a 'ProofExpr' of type
+-- 'ProofTriple', but contains the 'PE.StatePred' type that the verifier uses.
```

Comment:
```suggestion
-- 'ProofTriple', but contains the 'PE.StatePredSpec' type that the verifier uses.
```

Reply:
It's probably more accurate to say that `toplevelTriple` was previously strange as a result of trying to work around the odd structure of `EquivTriple`, and now it's a bit more straightforward.